meta-facebook/yv3-vf/prj.conf

```diff
@@ -0,0 +1,49 @@
+# cmsis-os2
+CONFIG_CMSIS_RTOS_V2=y
+CONFIG_NUM_PREEMPT_PRIORITIES=56
+CONFIG_HEAP_MEM_POOL_SIZE=32768
+CONFIG_CMSIS_V2_MSGQ_MAX_DYNAMIC_SIZE=8192
+CONFIG_POLL=y
+CONFIG_THREAD_NAME=y
+CONFIG_THREAD_STACK_INFO=y
+CONFIG_THREAD_MONITOR=y
+CONFIG_IRQ_OFFLOAD=y
+CONFIG_INIT_STACKS=y
+CONFIG_MAX_THREAD_BYTES=4
```

Comment:
This requires userspace. so it can be removed.

Reply:
remove it.